contribute/design/proposal-cstor-orchestration.md

```diff
@@ -0,0 +1,36 @@
+# Managing cStor based OpenEBS Volumes. 
+
+## Background 
+
+
+## Design
+
+Provisioning of cStor based Volume involve two steps. 
+
+1. Creating cStor StoragePools:
+
+- Admin will create SPC that contains information like:
```

Comment:
If I'm not wrong, this seems like contains the pool properties. should we add raid or mirror properties of the pool also?
do we need to have pool name also?

Reply:
cool. added.